sickbeard/name_parser/regexes.py

```diff
@@ -187,7 +187,7 @@
      # 01 - Ep Name
      # 01 - Ep Name
      r'''
-     ^((?P<series_name>.+?)(?:[. _-]{2,}|[. _]))?    # Show_Name and separator
+     ^((\[(.*?)\])[. _-])?((?P<series_name>.+?)(?:[. _-]{2,}|[. _]))?    # Show_Name and separator
```

Comment:
Missing a * here. and two sets of extra parens, double check.
`(\[.*?\][. _-]*)?`

so 
`^(\[.*?\][. _-]*)?((?P<series_name>.+?)(?:[. _-]{2,}|[. _]))?`